src/settings.rs

```diff
@@ -71,13 +76,17 @@ impl Default for Settings {
             port: DEFAULT_PORT,
             host: "127.0.0.1".to_string(),
             database_url: "mysql://root@127.0.0.1/syncstorage".to_string(),
+            tokenserver_database_url: None,
             database_pool_max_size: None,
             database_pool_min_idle: None,
             #[cfg(test)]
             database_use_test_transactions: false,
             actix_keep_alive: None,
             limits: ServerLimits::default(),
             master_secret: Secrets::default(),
+            tokenserver_jwks_rsa_exponent: None,
```

Comment:
Future note:
I wonder if we should try to collect the various `tokenserver_*` stuff into their own struct? Just wondering if it might make dealing with them a bit easier, particularly with hand off to `get_sync`

Reply:
Yes, definitely, I mentioned that when we were talking voice. Seems like a good refactor.